### ISSUE TITLE
improve viewing of multiple reports

### ISSUE THREAD

**Issue by rebeccahawke** [MSLNZ/equipment-register-schema#22]
move the reportNumber element to be a mandatory attribute in the report element

**Comment by jborbely** [comment]
Should the same name `reportNumber` be used? It's the two words **report report** next to each other that I'm questioning.

Some examples for how folding would look in an IDE (e.g., PyCharm, Visual Studio) for different attribute names

```
<report reportNumber="something">...</report>
<report number="something">...</report>
<report id="something">...</report>
```

although, if the IDE uses an XML colouring scheme the words would stand apart better

```xml
<report reportNumber="something">...</report>
<report number="something">...</report>
<report id="something">...</report>
```

Should this attribute be added to a `digitalReport` element? I think so. Otherwise when someone is reviewing the XML document, it will be impossible to know the chronological order of calibrations unless the report is opened to read/view the appropriate metadata.

**Comment by jborbely** [comment]
Using `reportIssueDate` would guarantee chronological order for a person viewing the XML document, but chronological order wasn't the main goal for this change. For MSL reports, the number is already chronological aware. Reports from outside labs would probably also be incrementing some number for the report number.

**Comment by rebeccahawke** [comment]
I agree that this attribute should be added to a digitalReport element, and any other calibration entry type.

issueDate would resolve chronological order for a given probe (for example) but it's not a unique identifier - often the set of reports get issued on the same date.

"id" is fine - it's short and doesn't detract from the actual report number (which is often a code rather than just a number anyway). I think 'id' is a bit more general but still pretty clear as to what it's asking for.

**Comment by jborbely** [comment]
The particular 17025 clause that we are discussing in this issue is to satisfy

> ISO/IEC 17025:2017 Clause 7.8.2.1
   d) unique identification that all its components are recognized as a portion of a complete report and a clear identification of the end

So, using `id` makes sense. It should be a required attribute of a `report` and of a `digitalReport` element.

**Comment by jborbely** [comment]
Fixed in 586518e15a37935e3609c5b884601ba2138f7b79
